### DISCUSSION TITLE
Can I use custom weights?

### DISCUSSION THREAD

**Discussion by lavinejames** [phosphor-icons/homepage#660] [Ideas]
Hello!

@rektdeckard There's a website I'm currently working on where I want to use Phosphor icons at very large sizes (around 200x200px), but keep the weight of the icons to roughly 1px. Is my only option to download and use the raw SVG, or is there a way I can use the `<i/>` tag approach and control the weight somehow?

Thanks!
James

**Comment by rektdeckard** [comment]
Unfortunately our libraries based on SVG use the outlined versions, which means they do not retain the stroke information and can't just have the weight changed to an arbitrary value. I believe some browsers may be able to synthesize font weights using the `font-synthesis` property to do a best-guess at a bold or italic weight, but this again doesn't apply to making a custom, thinner weight and probably wouldn't look very good anyway.

Your best bet is to use the raw SVGs in our asset kit or Figma plugin/library, and resize them yourself. Sorry about that!
